zips/draft-coinholder-directed-grants.md

```diff
@@ -0,0 +1,192 @@
+4. **Administrative Safeguards** — The Administrator or its delegate SHALL manage proposal submission, eligibility screening, compliance checks, and quarterly reporting, including KYC for grants > 50,000 USD equivalent.  
+5. **Keyholder Oversight** — Key-Holder Organizations MUST execute approved disbursements unless vetoed under the conditions defined in ZIP 1016 (Veto Process).  
+6. **Legal Compliance** — All parties MUST comply with applicable legal, tax, and reporting obligations.  
+7. **Security and Custody** — Multisig keys MUST be secured per the standards in ZIP 1016.  
+
+---
+
+## Non-Requirements
+
+* Changes to governance or funding mechanisms of other programs are **outside scope**.  
+* This ZIP does not modify consensus parameters or introduce new Funding Streams beyond those defined in ZIP 1016.  
+
+---
+
+## Specification
+
+### 1. Program Structure
+
+The Coinholder Grants Program SHALL operate as a **retroactive funding mechanism** for the 12% Funding Stream defined in ZIP 1016.  
+
+Funds are held in the Coinholder-Controlled Fund multisig wallet and distributed per approved proposals.  
+
+The Administrator or its delegate is responsible for managing proposal intake, eligibility review, publication, and transparency reporting.  
+
+Key-Holder Organizations (ECC, ZF, Shielded Labs) execute multisig disbursements upon coinholder approval, subject to veto provisions.  
+
+---
+
+### 2. KYC and Compliance
+
+Grants > 50,000 USD require KYC verification of the recipient by the Administrator or its delegate prior to payment.  
+
+The Administrator or its delegate MAY charge a reasonable administrative fee to cover these duties, subject to coinholder approval.  
+
+---
+
+### 3. Participation and Voting Thresholds
+
+* Participation threshold: ≥ 420,000 ZEC voted (≈ 2% of total supply).  
+* Approval criterion: simple majority of “Yes” votes.  
+* Each proposal vote is independent; coins used in one vote MAY be reused in another.  
+
+---
+
+### 4. Voting Process
+
+The Coinholder Grants Program supports two voting methods: **Shielded Voting** and **Transparent Voting**.  
+
+Both MUST produce verifiable and auditable results administered by the Administrator or its delegate.  
+
+#### 4.1 Shielded Voting Process
+
+Shielded voting is conducted through the dedicated Coin Voting 2.0 application that uses Orchard addresses and the Halo2 proof system to verify balances and prevent double voting.  
+
+1. **Registration Period** – The Administrator or its delegate MUST define a registration window not shorter than 10 days, specified by block heights. Only Orchard notes created or refreshed within this window are eligible for voting.  
+2. **Pre-Vote Safeguards** – Participants SHOULD move funds out of the wallet used for voting *before* entering their seed phrase into the Coin Voting application to reduce risk if the third-party app is compromised.  
```

Comment:
Participants aren't bound by this spec, but maybe something about "the Administrators MUST provide clear guidance to participants ..."